modules/@ergo/core/components/List/ListSearchTreeSelectHeader.vue

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <ListHeader :header="header">
+        <TreeSelect
```

Comment:
It's copy of ListSearchSelectHeader - I would create here slot for Select and globally renamed this component to ListSearchLanguageHeader cuz it's related to navigate between languages.